packages/liveblocks-node/src/client.ts

```diff
@@ -1384,6 +1390,39 @@ export class Liveblocks {
     return (await res.json()) as Promise<ThreadParticipants>;
   }
 
+  /**
+   * Gets a thread's subscriptions.
+   *
+   * @param params.roomId The room ID to get the thread subscriptions from.
+   * @param params.threadId The thread ID to get the subscriptions from.
+   * @param options.signal (optional) An abort signal to cancel the request.
+   * @returns An array of subscriptions.
+   */
+  public async getThreadSubscriptions(
+    params: { roomId: string; threadId: string },
+    options?: RequestOptions
+  ): Promise<{ data: UserSubscriptionData[] }> {
```

Comment:
Will this method (ever) need pagination?

Reply:
We'll likely add it post-launch, and the `{ data }` shape doesn't require breaking changes